docs/5.plugins/3.plugin-nlu.mdx

```diff
@@ -31,28 +31,46 @@ The first step is to install the module with the following command in your bot's
 npm install @botonic/plugin-nlu
 ```
 
-This command will install all the necessary stuff to start working with your intents.
+This command will install all the necessary stuff to start working with your intents and entities.
 
 
 ### **Defining your intents**
 
 Once installed, you will need to define a new directory called `nlu` under the `src` folder of your bot. Inside this new folder, 
-we are going to create a subdirectory called `intents`, which will contain your multilingual intents. Then for every language you want to support, 
-you will need to create another directory under `src/nlu/intents` with its language code (it must be [ISO 639-3](https://iso639-3.sil.org/code_tables/639/data)).
+we are going to create a subdirectory called `utterances`, which will contain your multilingual intents. Then for every language you want to support, 
+you will need to create another directory under `src/nlu/utterances` with its language code (it must be [ISO 639-3](https://iso639-3.sil.org/code_tables/639/data)).
 After having defined this in your project, you just need to add a bunch of simple text files as `{YOUR_INTENT}.txt` and fill them with the utterances you think that best describe each intent. **Every sentence must be on a different line of the file.**
 
 For the sake of clarity, let's illustrate with an example how your directory should look at this point:
 
-**src/nlu/intents/eng**
+**src/nlu/utterances/eng**
 * BookRestaurant.txt
 * GetDirections.txt
 * Greetings.txt
 
-So in this example, we have added three different intents which our bot will know how to understand. Obviously, 
-but the more example utterances you have, the better results you will obtain.
+So in this example, we have added three different intents which our bot will know how to understand. Obviously, the more example utterances you have, the better results you will obtain.
```

Comment:
but it's the plugin which reads the files and not the bot, right?
In any case, why not just saying "in this example, we have added three different intents which our bot will know how to understand." You already described the format of the files.

Reply:
--> In this example, we have added three different intents which our bot will know how to understand.